tests/test_wave_05.py

```diff
@@ -52,13 +52,19 @@ def test_new_genre_rec_from_empty_friends():
             }
         ]
     }
+    # Act
+    recommendations = get_new_rec_by_genre(sonyas_data)
+
+    # Assert
+    assert len(recommendations) == 0
+    assert INTRIGUE_1b not in recommendations
```

Comment:
Nice act and assert! I think we can remove line 60. If the line 59 passes we know the `recommendations` list is empty, and thus `INTRIGUE_1b` cannot be in it.